client/source/sass/main/project/_layout.scss

```diff
@@ -86,6 +86,7 @@
 
   .data {
     height: 30px;
+    margin-right: 10px;
     &.bordered {
```

Comment:
Hey @guptaarun: that was page in the branch I was working on, but no worries, I have taken care of it now.